user_docs/en/userGuide.t2t

```diff
@@ -1220,6 +1220,19 @@ By default, NVDA will detect the formatting at the position of the System caret
 
 Enable this option while proof reading documents in applications such as Microsoft Word, where formatting is important.
 
+==== Report indents as ====
+When Report Line Indentation is enabled, the Report indents as combo box has three options.
+
+- Tones
+- Speech
+- Both Speech and Tones
+

```

Comment:
nit: Change this list ending to a dash on a line by itself rather than a blank line:

`-`

In changes.t2t, we use a double blank line because it's more natural when there are only lists, but in the User Guide, we use the dash method because lists are interspersed with other text, so it's clearer.